lib/cosmos/tools/tlm_viewer/tlm_viewer_config.rb

```diff
@@ -205,24 +213,31 @@ def save(filename)
 
       File.open(filename, 'w') do |file|
         @columns.each_with_index do |target_screen_infos, column_index|
-          if column_index != 0
-            file.puts ''
-            file.puts "NEW_COLUMN"
-            file.puts ''
-          end
+          file.puts "NEW_COLUMN\n\n" if column_index != 0
```

Comment:
The new lines caused by this are not equivalent.

Reply:
The old code didn't work with GROUP and GROUP_SCREEN so yes it is different.

Reply:
I was referring to the newlines being different.  Before it put one before and after the NEW_COLUMN

Reply:
I think the newline at 244 takes care of newlines before. If you manually add a puts config on line 390 in the spec you'll see how it looks.